doc/nrf/releases_and_maturity/migration/nRF54H20_migration_2.7/migration_guide_2.6.99-cs2_to_2_7_environment.rst

```diff
@@ -0,0 +1,255 @@
+2_6_99_cs2:orphan:
+
+.. _migration_cs3_to_2_7_env:
+
+Update your development environment for |NCS| v2.7 (for v2.6.99-cs2 users)
+##########################################################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+This document describes how to update your development environment for the nRF54H20 DK from |NCS| v2.6.99-cs2 to |NCS| v2.7.
+
+The main development environment changes introduced by 2.7 for the nRF54H20 DK are the following:
+
+..
+   ##TODO
+
+Minimum requirements
+********************
+
+Make sure you have all the required hardware, software, and that your computer has one of the supported operating systems.
+
+Hardware
+========
+
+* nRF54H20 DK version PCA10175 v0.7.x or v0.8.0 (ES3), ES4.
+  These are the only versions of the nRF54H20 DK compatible with |NCS| 2.7.
+  Check the version number on your DK's sticker to verify its compatibility with |NCS| version 2.7.
+* USB-C cable.
+
+Software
+========
+
+On your computer, one of the following operating systems:
+
+.. include:: ../../../../nrf/installation/recommended_versions.rst
+    :start-after: os_table_start
+    :end-before: os_table_end
+
+See :ref:`supported_OS` for more information about the tier definitions.
+
+|supported OS|
+
+You also need the following:
+
+* `Git`_ or `Git for Windows`_ (on Linux and Mac, or Windows, respectively)
+* `curl`_
+* On Windows, SEGGER USB Driver for J-Link
+* SEGGER `J-Link version 7.94e`_
+
+.. note::
+   The SEGGER USB Driver for J-Link was included, on Windows, in the nRF Command Line Tools bundle required by the |NCS| v2.4.99-cs3.
+   When updating SEGGER J-Link to version 7.94e on Windows, select :guilabel:`Update existing installation` under :guilabel:`Choose destination` in the :guilabel:`choose optional components` window.
+
+   See the following screenshot:
+
+      .. figure:: images/jlink794e_install.png
+         :alt: Optional components in the J-Link installation
+
+Preliminary step
```

Comment:
This shouldn't be required for this migration. West update should work if tag  for nrf will be changed.